public/manifest.json

```diff
@@ -4,12 +4,12 @@
   "icons": [
     {
       "src": "favicon.ico",
-      "sizes": "192x192",
```

Comment:
Looks like maybe you are forgetting up update your local branch before starting on other work or working off the wrong branch?  Means that work items are getting mixed.  This one was already merged back into master...